graphs/load-balancer.fbp

```diff
@@ -0,0 +1,37 @@
+# load-balancer.fbp
+
+# Sends the input packet to one of the target URLs listed in the file referenced
+# in the environment variable whos name is sent to TARGET_LIST_FILE_ENV.
```

Comment:
For cross platform portability. This would allow a system administrator to move/copy the deployment without modifying the pipeline parameters. Very useful when deploying between staging and production machines.